app/assets/stylesheets/bootstrap_and_overrides.css

```diff
@@ -1,15 +1,28 @@
 /*
-  =require twitter-bootstrap-static/bootstrap
+=require twitter-bootstrap-static/bootstrap
 
-  Use Font Awesome icons (default)
-  To use Glyphicons sprites instead of Font Awesome, replace with "require twitter-bootstrap-static/sprites"
-  =require twitter-bootstrap-static/fontawesome
-  */
+Use Font Awesome icons (default)
+To use Glyphicons sprites instead of Font Awesome, replace with "require twitter-bootstrap-static/sprites"
+=require twitter-bootstrap-static/fontawesome
+*/
 
-  td, th {
-    padding: 0 2rem;
-  }
+td, th {
+  padding: 0 2rem;
+}
 
-  table {
-    border: 1px solid rgb(221, 215, 215);
-  }
+table {
+  border: 1px solid rgb(221, 215, 215);
+}
+
+.form-container {
+  display: flex;
+}
+
+.form-group{
+  display: flex;
+  flex-direction: column;
+}
+
+.form-group > select {
+  margin: 5px 0;
+}
```

Comment:
These new styles look like they could go in their own stylesheet and imorted into `application.css` rather than as boostrap overrides.